malpolon/models/standard_prediction_systems.py

```diff
@@ -160,14 +163,18 @@ def _step(
         y_hat = self(x)
 
         loss = self.loss(y_hat, self._cast_type_to_loss(y))  # Shape mismatch for binary: need to 'y = y.unsqueeze(1)' (or use .reshape(2)) to cast from [2] to [2,1] and cast y to float with .float()
-        self.log(f"loss/{split}", loss, **log_kwargs)
+        self.log(f"loss_{split}", loss, **log_kwargs)
 
         for metric_name, metric_func in self.metrics.items():
             if isinstance(metric_func, dict):
-                score = metric_func['callable'](y_hat, y, **metric_func['kwargs'])
+                if metric_func['kwargs']:
```

Comment:
Indeed, metric kwargs are expected to be added in the config file; however, when the `kwargs` is empty, calling metric_func with kwargs will trigger a None-type error.

I suggest you replace that if condition by the following line instead:
`score = metric_func['callable'](y_hat, y, metric_func.get('kwargs', {}))`
replacing the old line 167.

This is a good addition, thx!